### DISCUSSION TITLE
Link Github Copilot with my Codespaces

### DISCUSSION THREAD

**Discussion by Chri790** [microsoft/SATechnicalOnboarding#120] [Questions]
Good day!
I am almost done with my Technical Onboarding. I am having troubles linking github copilot with my codespace - it keeps saying "**Not signed in with github copilot, but only signed in with my gihub account**

I'm not sure how to go about it. Please assist

**Comment by AnthonyByansi** [comment]
Hi @Chri790 
To utilze GitHub Copilot, you need to be an authenticated `GitHub Student Developer`  Please follow the following [Step-by-Step guide](https://techcommunity.microsoft.com/t5/educator-developer-blog/step-by-step-setting-up-github-student-and-github-copilot-as-an/ba-p/3736279?WT.mc_id=%3Fwt.mc_id%3Dstudentamb_260352) to set Up GitHub Student and GitHub Copilot

**Reply by Chri790** [reply]
Thank you so much!

I already have it. How then can I link it with my Student Ambassador Github account?

**Reply by Chri790** [reply]
Also, I keep getting this error:
**You have divergent branches and need to specify how to reconcile them.**

How do I fix it?

**Comment by AnthonyByansi** [comment]
> Thank you so much!
> 
> I already have it. How then can I link it with my Student Ambassador Github account?

Hi @Chri790 Your students GitHub copilot can only be linked to one GitHub account. So, if you redeemed your access to GitHub copilot with your personal account, you can't redeem it once again with your student ambassador GitHub account.

If you wish to try out how copilot works, switch back to the account that you used to redeem access to GitHUb Copilot.

**Comment by AnthonyByansi** [comment]
> Also, I keep getting this error: **You have divergent branches and need to specify how to reconcile them.**
> 
> How do I fix it?

This indicates that you ran into a merge conflict. I can help on a quick call via teams!

**Reply by Chri790** [reply]
Alright thanks!

You can give the call when you have time. As for me, on my side we are 10:43am, I am free now, but will be busy at 12pm.
So, now is fine with me

**Reply by AnthonyByansi** [reply]
What could be your teams alias? so I could have you contacted

**Reply by Chri790** [reply]
I will text you on Teams after 12pm if that is fine with you
I know you on Teams

**Reply by Chri790** [reply]
I texted you already on Teams

You can call when you are ready

Thank you
